Testing/PythonTests/toImage.py

```diff
@@ -13,11 +13,13 @@ def toImageTest1():
 val = toImageTest1()
 
 if val is False:
+  print("toImageTest1 failed")
   sys.exit(1)
 
 def toImageTest2():
   mesh = Mesh(os.environ["DATA"] + "/femur.ply")
-  img = mesh.toImage(spacing=[2.0, 2.0, 1.0])
+  bbox = mesh.boundingBox() # changing spacing means we need to pad more to compensate
```

Comment:
Unfortunately... I wonder if we could interpret this padding argument in the to be image space since the user very likely wants the image in the end to have a logical padding added.

Reply:
Removed padding from these two function. If user wants a larger region of the mesh they can make the region larger (added a function to PhysicalRegion to do this). If they want a number of pixels on the boundary they can call Image::pad on the result. Updated all tests and examples and Python bindings.